src/resmom_win/start_exec.c

```diff
@@ -4479,20 +4364,12 @@ std_file_name(job *pjob, enum job_file which, int *keeping)
 		}
 
 		pd = strrchr(pjob->ji_wattr[(int)JOB_ATR_jobname].
-#ifdef WIN32
 			at_val.at_str, '\\');
-#else
-			at_val.at_str, '/');
-#endif
 
 		if (pd == NULL) {
 			pd = pjob->ji_wattr[(int)JOB_ATR_jobname].
 				at_val.at_str;
-#ifdef WIN32
 			(void)strcat(path, "\\");
-#else
-			(void)strcat(path, "/");
-#endif
 		}
 
 		(void)strcat(path, pd);		/* start with the job name */
```

Comment:
Please log the error :
https://github.com/PBSPro/pbspro/blob/06f24712f1b101a386d5b88656a24489250e52f0/src/resmom_win/start_exec.c#L4466